CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # CHANGELOG
 
+## v0.6.2
+
+This minor release of Hermes re-enables the `upgrade client`,
```

Comment:
One last comment:

I see that the content of .changelog/v0.6.2/summary.md differs from this. Maybe we need to regenerate the changelog.